src/views/home/HomeViewProvider.ts

```diff
@@ -50,12 +51,14 @@ export class HomeViewProvider implements vscode.WebviewViewProvider {
           const organizationsController = new OrganizationsCLIController(folder)
           await organizationsController.selectOrganization(data.value, false)
           await executeRefreshUsagesCommand(folder)
+          await executeRefreshInspectorCommand()
```

Comment:
There's [refreshAllCommand](https://github.com/DevCycleHQ/vscode-extension/blob/main/src/extension.ts#L105) that refreshes all the providers it's registered with. You should be able to use that instead of making a new refresh command. Also should be able to get rid of the `executeRefreshUsagesCommand` right above these.

Reply:
oh nice, fixed! i removed the whole refreshInspector folder since the command isn't used anywhere now